lib/Backend/Inline.cpp

```diff
@@ -4667,6 +4667,10 @@ Inline::MapFormals(Func *inlinee,
             {
                 break;
             }
+            if (instr->m_func != inlinee)
```

Comment:
The check only verifies if the ArgIn_rest belongs to the inline or the inner inlinee.